basic-application/basic-application-core/src/main/java/fr/openwide/core/basicapp/core/business/referential/model/search/AbstractGenericListItemSearchQueryImpl.java

```diff
@@ -0,0 +1,36 @@
+package fr.openwide.core.basicapp.core.business.referential.model.search;
+import org.apache.lucene.search.SortField;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import fr.openwide.core.basicapp.core.util.binding.Bindings;
+import fr.openwide.core.jpa.more.business.generic.model.EnabledFilter;
+import fr.openwide.core.jpa.more.business.generic.model.GenericListItem;
+import fr.openwide.core.jpa.more.business.search.query.AbstractHibernateSearchSearchQuery;
+import fr.openwide.core.jpa.more.business.sort.ISort;
+
+@Component
+@Scope("prototype")
```

Comment:
I also think this class (and the corresponding interface) could be pushed up to owsi-core-components-jpa-more, in package fr.openwide.core.jpa.more.business.generic.query.